dashboard/src/modules/components/TableOfContent/index.jsx

```diff
@@ -207,171 +209,173 @@ const TableOfContent = () => {
-                        </Tr>
-                      ))
-                    ) : (
-                      <Tr>
-                        <Td colSpan={8}>
-                          <EmptyTable />
-                        </Td>
-                      </Tr>
-                    )}
-                  </Tbody>
-                </>
-              )}
-            </TableComposable>
-            <TablePagination
-              numberOfRows={tableData.length}
-              page={page}
-              setPage={setPage}
-              perPage={perPage}
-              setPerPage={setPerPage}
-            />
+                        })}
+                      </DrilldownMenu>
+                    }
+                  >
+                    {data.name}
+                  </MenuItem>
+                );
+              })}
+              {contentData?.files?.map((data, index) => {
+                return (
+                  <MenuItem
+                    key={index}
+                    onClick={() => {
+                      updateHighlightedRow(index);
+                    }}
+                  >
+                    {data.name}
+                  </MenuItem>
+                );
+              })}
+            </MenuList>
+          </MenuContent>
+        </Menu>
+        <div className="tableTOC">
+          <div className="searchTOCContainer">
+            <SearchTOC
+              dataArray={searchSpace}
+              setTableData={setTableData}
+            ></SearchTOC>
           </div>
+          <TableComposable
+            aria-label="Simple table"
+            variant="compact"
+            className="tocBody"
+          >
+            <>
+              <Thead>
+                <Tr>
+                  <Th>Name</Th>
+                  <Th>mtime</Th>
+                  <Th>Size</Th>
+                  <Th>Mode</Th>
+                  <Th>Type</Th>
+                  <Th></Th>
+                </Tr>
+              </Thead>
+              <Tbody>
+                {visibleTableFiles.length > 0 ? (
+                  visibleTableFiles?.map((file, index) => (
+                    <Tr
+                      key={file.name}
+                      className={
+                        activeFile === index + (page - 1) * perPage
+                          ? "active"
+                          : ""
+                      }
+                    >
+                      <Td dataLabel={file.name}>{file.name}</Td>
+                      <Td dataLabel={file.mtime}>{file.mtime}</Td>
+                      <Td dataLabel={file.size}>{file.size}</Td>
+                      <Td dataLabel={file.mode}>{file.mode}</Td>
+                      <Td dataLabel={file.type}>{file.type}</Td>
+                      <Td dataLabel={file.uri}>
+                        <a
+                          className="download-icon"
+                          href={file.uri}
+                          target="_blank"
+                          rel="noreferrer"
```

Comment:
This is cool, and I think reveals a bug in the server 😆 

I'm getting weird downloaded file names like `fio_rw_2018.02.01T22.40.57_metadata.log`, which I suspect comes from the fact that our `Content-Disposition` header is specifying the file *path* `fio_rw_2018.02.01T22.40.57/metadata.log` instead of just the name.